### PR TITLE
Added Groups/Hosts/Sources count to inventory list rows #7674

### PR DESCRIPTION
**Pull request by asohh**
Hey, 

I implemented the feature described in https://github.com/ansible/awx/issues/7674.

### CODE REVIEW

**Comment by unlikelyzero** [comment]
Note to QE Should look at responsivity

File: awx/ui_next/src/screens/Inventory/InventoryList/InventoryListItem.jsx [hunk 1]
```diff
@@ -97,11 +110,33 @@ function InventoryListItem({
                 </Link>
               )}
             </DataListCell>,
+
```
Comment (AlexSCorey) [inline awx/ui_next/src/screens/Inventory/InventoryList/InventoryListItem.jsx]: We can remove this extra line

File: awx/ui_next/src/screens/Inventory/InventoryList/InventoryListItem.jsx [hunk 2]
```diff
@@ -29,6 +30,18 @@ const DataListAction = styled(_DataListAction)`
...
 `;
 
+const Badge = styled(PFBadge)`
+  margin-left: 8px;
+`;
+
+const ListGroup = styled.span`
+  margin-left: 24px;
+
+  &:first-of-type {
```
Comment (AlexSCorey) [inline awx/ui_next/src/screens/Inventory/InventoryList/InventoryListItem.jsx]: This bit does not seem to be doing anything and I think we can remove it

**Review by AlexSCorey — CHANGES_REQUESTED** [review]
@asohh Thanks for doing this. There are just a few changes to be made. Please make those changes, and then fix the failing ui_nest tests by running prettier (`npm run prettier`) and this should be ready.  Let me know if you have questions.

File: awx/ui_next/src/screens/Inventory/InventoryList/InventoryListItem.jsx (hunk 1 above)
Comment (asohh) [reply awx/ui_next/src/screens/Inventory/InventoryList/InventoryListItem.jsx]: Removed this and another extra line

File: awx/ui_next/src/screens/Inventory/InventoryList/InventoryListItem.jsx (hunk 2 above)
Comment (asohh) [reply awx/ui_next/src/screens/Inventory/InventoryList/InventoryListItem.jsx]: Thank you, removed that part.

**Comment by asohh** [comment]
> @asohh Thanks for doing this. There are just a few changes to be made. Please make those changes, and then fix the failing ui_nest tests by running prettier (`npm run prettier`) and this should be ready. Let me know if you have questions.

Thank you. I changed the lines and ran `npm run prettier`

**Review by AlexSCorey — APPROVED** [review]
This looks good!

**Comment by tiagodread** [comment]
@asohh great job!  I found a few detail related to responsivity:

From 879px to 1685px (menu opened)  the count block breaks the text from counter to a new line even "having available" space on the row, see

counter

**Comment by asohh** [comment]
Hey @tiagodread 
I changed the ListGroup containing the text and count into a div with the display type of inline-block.

Now it's not splitting them anymore.

**Review by tiagodread — APPROVED** [review]
Great job!

**Pull request merged**